cmd/awat/import.go

```diff
@@ -80,10 +80,8 @@ var getImportByIDCmd = &cobra.Command{
 
 		if status == nil {
 			fmt.Printf("No Import found with ID %s", imprt)
-		}
-
-		if status != nil {
-			_ = printJSON(status)
+		} else {
+			printJSON(status)
 		}
 
 		return nil
```

Comment:
Not sure if error from `printJSON` should be ignored.
```suggestion
			return nil
		}
		return printJSON(status)
```